installer/scripts/uninstall.sh

```diff
@@ -67,15 +67,33 @@ check_if_pkg_is_installed() {
 # Do our stuff - uninstall the OMS agent
 
 pkg_rm omsconfig
-pkg_rm omsagent
 
-# If MDSD is installed and we're just removing (not purging), leave SCX
-
-MDSD_INSTALLED=1
+# If MDSD/LAD is installed and we're just removing (not purging), leave OMS, SCX and OMI
```

Comment:
why is this code duplicated?

Reply:
I don't know, it was like that. @simathih probably knows the rason behind that.

Reply:
could be one is called from shell bundle and another from omsadmin.sh....I am not 100% sure though

Reply:
nop, I just figured out that the duplication is important if the "uninstall.sh" is not there, like in old agents.